projects/robots/boston_dynamics/spot/protos/Spot.proto

```diff
@@ -0,0 +1,1459 @@
+#VRML_SIM R2020a utf8
+# license: Creative Commons 4.0
+# license url: https://creativecommons.org/licenses/by/4.0
+# documentation url: https://www.cyberbotics.com/doc/guide/spot
+# The "Spot" is a dog robot developed by Boston Dynamics.
+# Original model by Greg McKechnie.
+
+PROTO Spot [
+  field SFVec3f     translation      0 -1.028 0.46             # Is `Transform.translation`.
+  field SFRotation  rotation         0 1 0 0                   # Is `Transform.rotation`.
+  field SFString    name             "Spot"                    # Is `Solid.name`.
+  field SFString    model            "Boston Dynamics - Spot"  # Is `Robot.model`.
+  field SFString    controller       "spot_moving_demo"        # Is `Robot.controller`.
+  field SFString    controllerArgs   ""                        # Is `Robot.controllerArgs`.
+  field SFString    customData       ""                        # Is `Robot.customData`.
+  field SFBool      supervisor       FALSE                     # Is `Robot.supervisor`.
+  field SFBool      synchronization  TRUE                      # Is `Robot.synchronization`.
+  field SFBool      selfCollision    FALSE                     # Is `Robot.selfCollision`.
+  field MFNode      frontExtension   []                        # Extends the robot with new nodes in the front shoulder part.
+  field MFNode      middleExtension  []                        # Extends the robot with new nodes in the middle back part.
+  field MFNode      rearExtension    []                        # Extends the robot with new nodes in the rear shoulder part.
+]
+{
+  Robot {
+    translation IS translation
+    rotation IS rotation
+    children [
+      Accelerometer {
+      }
+      GPS {
+      }
+      Gyro {
+      }
+      InertialUnit {
+      }
+      LED {
+        translation -0.0835156 1.67783 -0.953027
+        rotation 0.4450559604323016 0.87360692233198 -0.19681498250216475 -0.9372053071795863
+        children [
+          Shape {
+            appearance GlossyPaint {
+              baseColor 0 1 0
+            }
+            geometry Box {
+              size 0.005 0.06 0.005
+            }
+          }
+        ]
+        name "left led"
+        color [
+          0 1 0
+        ]
+      }
+      LED {
+        translation 0.0835156 1.67783 -0.953027
+        rotation -0.4197661711038622 0.8854393609202096 -0.19948308131271172 0.985158
+        children [
+          Shape {
+            appearance GlossyPaint {
+              baseColor 0 1 0
+            }
+            geometry Box {
+              size 0.005 0.06 0.005
+            }
+          }
+        ]
+        name "right led"
+        color [
+          0 1 0

```

Comment:
About this color, did you find it somewhere in video or datasheet that it can be green only?

Reply:
In all the videos and images I found, it was always green. I don't think these leds have any other function than to indicate that Spot is on and running. Maybe it could be red to indicate that Spot has run out of battery power... I don't know. Anyway, if it was to make light, there would be some around the other cameras as well.